### DISCUSSION TITLE
Usage with CircleCI

### DISCUSSION THREAD

**Discussion by gabrielsch** [vercel/turborepo#538] [Help] [answered]
How does Turborepo pipelines compares with CirceCI job steps parallelization?

CircleCI has its own way for parallelizing tasks and cache previous steps arctifacts, how it should be set when its used with Turborepo?

Should we run the pipeline in a single CircleCI step? That means Circle won't need to spin up a new machine for an upcoming step, right?

Thanks for the great work! I'm doing some research here and checking if migrating to Turborepo will speed up our CI/CD.

**Comment by anthonyshew** [comment, marked as answer]
CI integrations can be found here!

https://turbo.build/repo/docs/ci
